css/style.css

```diff
@@ -60,6 +60,47 @@ form li {
   transition: transform 0.2s;
 }
 
+.m-menu-continer {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 700px;
+  width: 100%;
+  background-color: rgb(96, 112, 255, 0.9);
+}

```

Comment:
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

 - [ ] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 





-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
|![20badmobile](https://user-images.githubusercontent.com/79058364/174662399-f65953ab-e328-46d7-aab8-7f1a869700bf.png) | ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |